Brojd/update insights docs (#3692)
grafana/oncall: docs/sources/insights-and-metrics/_index.md

Be aware: if you have made changes to the dashboard, they will be lost after re-importing or after the plugin update.
To save your changes go to the "OnCall Metrics" dashboard settings, click "Save as" and save a copy of the dashboard.

> The most useful insights are also present within Grafana OnCall itself. To see them follow the steps:
> 
> 1. Open Grafana OnCall.
> 2. Click **Insights** sub-section in the left-side menu.

Reviewer: ```suggestion
You can also view Insights from Grafana OnCall. 

To view Insights, complete the following steps.

1. Open Grafana OnCall.
2. Click the **Insights** sub-section in the navigation menu.
```
